src/scripts.js

```diff
@@ -5,58 +5,47 @@ import './images/travel-tracker-logo.webp';
 import './images/profile.webp'
 
 // Import local files
-import { fetchData, fetchAll, postData } from './apiCalls.js';
+import { fetchData, fetchAll, postTripRequest } from './apiCalls.js';
 import Traveler from './travelers/Traveler';
-import Destination from './destinations/Destination';
+// import Destination from './destinations/Destination';
 import Trip from './trips/Trip';
 import DestinationsRepo from './destinations/DestinationsRepo';
 
 // Global variables
-const isBetween = require('dayjs/plugin/isBetween');
-// let allTravelers;
-let allDestinations
-// let allTrips;
+// const isBetween = require('dayjs/plugin/isBetween');
+let allDestinations;
 let randomUser;
 
 
 // Import third party libraries
-import dayjs from 'dayjs';
-import customParseFormat from 'dayjs/plugin/customParseFormat';
-dayjs.extend(isBetween);
-dayjs.extend(customParseFormat);
+// import dayjs from 'dayjs';
+// import customParseFormat from 'dayjs/plugin/customParseFormat';
+// dayjs.extend(isBetween);
+// dayjs.extend(customParseFormat);
 import MicroModal from 'micromodal';  
 
 // Query Selectors
-const username = document.querySelector('#username');
-const userSpending = document.querySelector('#userSpending');
-// const travelerType = document.querySelector('#travelerType');
-const pastTrips = document.querySelector('#pastTrips');
-const upcomingTrips = document.querySelector('#upcomingTrips');
-const pendingTrips = document.querySelector('#pendingTrips');
-const formDestinations = document.querySelector('#formDestinations');
+const username = document.getElementById('username');
+const userSpending = document.getElementById('userSpending');
+const pastTrips = document.getElementById('pastTrips');
+const upcomingTrips = document.getElementById('upcomingTrips');
+const pendingTrips = document.getElementById('pendingTrips');
+const formTitle = document.getElementById('modalFormTitle');
+const formDestinations = document.getElementById('formDestinations');
+const submitForm = document.getElementById('requestForm');
+const estimatedCost = document.getElementById('formEstimatedCost');
+const getEstimatedCost = document.getElementById('getEstimatedCost');
+const formTripDate = document.querySelector('#formTripDate');
+const formDestination = document.querySelector('#formDestinations');
+const formDuration = document.querySelector('#formDuration');
+const formNumOfTravelers = document.querySelector('#formNumOfTravelers');
 
 // Event Listeners
-window.addEventListener('load', renderApplication);
-
-
-// Functions
-// function getData() {
-//   fetchAll()
-//   .then(data => {
-//     const destinationsData = data;
-//     allTrips = new TripsRepo(tripsData.trips);
-//     allTravelers = new TravelersRepo(travelersData.travelers, allTrips);
-//     allDestinations = new DestinationsRepo(destinationsData.destinations);
```

Comment:
Why was this strategy abandoned and instead pieced out into each separate render function below?